packages/lesswrong/components/ea-forum/users/EAUsersProfile.tsx

```diff
@@ -305,33 +305,33 @@ const EAUsersProfile = ({terms, slug, classes}: {
 
   const privateSectionTabs: Array<UserProfileTabType> = [{
     id: 'drafts',
-    label: `${ownPage ? 'My' : `${username}'s`} Drafts`,
+    label: `Drafts`,
```

Comment:
IMO the changes in this file could have been a separate PR

Reply:
Fair